go/connector.go

```diff
@@ -23,18 +23,17 @@ package athenadriver
 import (
 	"context"
 	"database/sql/driver"
-
+	credentials2 "github.com/aws/aws-sdk-go-v2/credentials"
```

Comment:
Cn this just be `"github.com/aws/aws-sdk-go-v2/credentials"`?

Reply:
Yes indeed! Fixed.